src/mscorlib/src/System/Threading/ManualResetEventSlim.cs

```diff
@@ -185,7 +185,7 @@ public ManualResetEventSlim()
 
         /// <summary>
         /// Initializes a new instance of the <see cref="ManualResetEventSlim"/>
-        /// class with a Boolen value indicating whether to set the intial state to signaled.
+        /// class with a Boolean value indicating whether to set the initial state to signaled.
```

Comment:
i suggest lower case boolean

Reply:
I'd say uppercase, but maybe I'm just old-fashioned on that one.